lib/sdam/cursor.js

```diff
@@ -91,13 +91,15 @@ class Cursor {
       transforms: options.transforms
     };
 
+    if (typeof options.session === 'object') {
```

Comment:
YES! This has been bugging me for forever! Can we incorporate this into the non-SDAM cursor?

Reply:
sure